mobile_config/src/boosted_hex_info.rs

```diff
@@ -93,18 +100,78 @@ impl BoostedHexInfo {
                 .num_seconds()
                 .checked_div(self.period_length.num_seconds())
                 .unwrap_or(0) as usize;
-            Ok(Some(self.multipliers[index]))
+            Some(self.multipliers[index])
         } else {
             // start time has not been previously set, assume this is the first time rewarding this hex
             // and use the first multiplier
-            Ok(Some(self.multipliers[0]))
+            Some(self.multipliers[0])
         }
     }
+
+    fn matches_device_type(&self, device_type: &BoostedHexDeviceType) -> bool {
+        self.device_type == *device_type || self.device_type == BoostedHexDeviceType::All
+    }
 }
 
-#[derive(Debug, Clone, Default)]
-pub struct BoostedHexes {
-    pub hexes: HashMap<Cell, BoostedHexInfo>,
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum BoostedHexDeviceType {
```

Comment:
Good point. I pulled `BoostedHexDeviceType` through the system before updating the proto definition. This was a remnant of that.

The change was showing up here because this branch was not properly rebased onto it's parent.

I've made the change in the HIP-109 PR, https://github.com/helium/oracles/pull/798/commits/90a7aa2242a9bb5cfffa581d1a7b8f85f0535764